jib-gradle-plugin/build.gradle

```diff
@@ -1,11 +1,24 @@
 plugins {
+    // Gradle API
     id 'java-gradle-plugin'
+
+    // Checkstyle
```

Comment:
Are all these comment necessary?

Reply:
Not really, I'll remove the obvious ones.